samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -390,7 +390,7 @@ For examples, see the related minimal overlays in the :ref:`nrf_cloud_multi_serv
 Customizing GNSS antenna configuration
 ======================================
 
-This sample uses the :ref:`lib_modem_antenna` library, which is enabled by default for builds targeting the ``nrf9161dk_nrf9161_ns``, ``nrf9160dk_nrf9160_ns``, or ``thingy91_nrf9160_ns`` board names.
+This sample uses the :ref:`lib_modem_antenna` library, which is enabled by default for builds targeting the ``nrf9151dk_nrf9151_ns``, ``nrf9161dk_nrf9161_ns``, ``nrf9160dk_nrf9160_ns``, or ``thingy91_nrf9160_ns`` board names.
```

Comment:
Should this line and Line 289 be worded more similarly? Both now use different wording, as well as Line 289 specifiying the device in addition to the build target.

Reply:
Agreed. Updated the doc.